changelog.d/12602.misc

```diff
@@ -0,0 +1 @@
+Add link to documentation in Grafana Dashboard
```

Comment:
Looks like the CI check is failing because there is no punctuation here. 
```suggestion
Add link to documentation in Grafana Dashboard.
```

Reply:
Was fixed in a separate commit, since it's requested in the contribution guide, to not do force pushes.